src/main/kotlin/view/LandMineMapUI.kt

```diff
@@ -0,0 +1,26 @@
+package view
+
+import domain.Line
+import dto.LandMineMapUIRequest
+
+class LandMineMapUI(private val landMineMapUIRequest: LandMineMapUIRequest) {
+
+    fun title() {
+        println()
+        println("지뢰찾기 게임 시작")
+    }
+
+    fun map() {
+        landMineMapUIRequest.mapCoordinates.map {
+            println()
+            display(it)
+        }
+    }
+
+    private fun display(mapLine: Line) {
+        mapLine.coordinates.forEach {
+            if (landMineMapUIRequest.landMineCoordinates.contains(it)) print("* ")
+            else print("C ")
+        }
+    }
```

Comment:
I/O 는 굉장히 큰 비용이 들어간다는 사실을 알고 계실텐데요.
I/O 비용을 줄여보면서 코틀린스럽게 코드를 작성해본다면 아래와 같이 표현할 수 있습니다.

```suggestion
    fun title() = with(StringBuilder()) {
        appendLine()
        appendLine("지뢰찾기 게임 시작")
        println(this)
    }

    fun map() = with(StringBuilder()){
        landMineMapUIRequest.mapCoordinates.map {
            appendLine()
            append(buildDisplay(it))
        }
        println(this)
    }

    private fun buildDisplay(mapLine: Line)= buildString {
         mapLine.coordinates.map {
            if (landMineMapUIRequest.landMineCoordinates.contains(it)){
                append("* ")
            }
            else {
                append("C ")
            }
        }
    }
```

Reply:
그리고 이 부분도 프로그래밍 요구사항을 참고하여 indent를 줄여주세요~